components/selects/SupportedNetworkSelect.tsx

```diff
@@ -63,8 +62,10 @@ export const SupportedNetworkSelect = forwardRef<
               disabled={disabledChainIds?.includes(tn.chainId)}
             >
               {tn.chainName} ({tn.symbol})
-              {deprecatedChains.includes(tn.chainId) && " - Deprecated"}
-              {disabledChainIds?.includes(tn.chainId)
+              {deprecatedChains.includes(tn.chainId as SUPPORTED_CHAIN_ID) &&
```

Comment:
I'd prefer to move this to a separate deprecated option group below, then we remove it fully after the deprecation date.